modules/ROOT/partials/configuration/width.adoc

```diff
@@ -7,6 +7,7 @@ NOTE: {productname} sets the width in pixels if a number is provided. However, i
 
 *Type:* `Number` or `String`
 
+[discrete]
```

Comment:
I don't think this should be used, no other page uses this discrete attribute. Other pages have the examples show up in the ToC.